### PR TITLE
refactor: edited the pull_request_template.md

### LINKED ISSUE #594: Update the Pull Request Template
> This issue is a part of `Hacktoberfest 2021`

Pull request is the most useful thing for making a great PR.

## What to do
- The current PR template is so simple and doesn't carry a lot of information
- The Job of this issue is to update the Pull Request Template.
- Update the pull request template to make it better, and to carry more information.

## Where to do
- Update the `.github/pull_request_template.md` file

## Before you contribute
- Make sure to read the [README.md](https://github.com/react-auth-kit/react-auth-kit/blob/master/README.md) file.
- Make sure you read the [CONTRIBUTING.MD](https://github.com/react-auth-kit/react-auth-kit/blob/master/CONTRIBUTING.md) file.

## Important ‼
**_Make sure your commits follow the  Conventional commit https://www.conventionalcommits.org/en/v1.0.0/, or it'll break the CI._**

_** The best way to avoid the linting issue in CI is always to set up your dev env with `npm install`. It will set husky and list stages._

> ***Need help?***
***Feel free to contact me @ [in2arkadipb13@gmail.com](mailto:in2arkadipb13@gmail.com)***

### PR DESCRIPTION
**Pull request by jason1985**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Description

Update to the `pull_request_template.md` that adds more details.
Please let me know if this format is ok or if anything needs to change.
Thank you!

## Related Issues

Fixes #594 

## Added tests?

- [ ] yes
- [x] no, because they aren't needed

## Added to documentation?

- [ ] readme
- [ ] contributing.md
- [x] docs
- [ ] no documentation needed

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,3 +9,29 @@
...
+-->
+
+## Screenshots/Recordings
+
+<!--  screenshots  -->
+
+## Added tests?
+
+- [ ] yes
+- [ ] no, because they aren't needed
```
Comment (darkmatter18) [inline .github/pull_request_template.md:30]: Please Capitalize the first characters.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -9,3 +9,29 @@
...
+
+## Added to documentation?
+
+- [ ] readme
+- [ ] contributing.md
+- [ ] docs
+- [ ] no documentation needed
```
Comment (darkmatter18) [inline .github/pull_request_template.md:37]: 1. redme change to README
2. contributing is not needed
3. Docs change to Documentation
4. Change to No documentation Needed

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -9,3 +9,29 @@
...
+
+## Description
+
+<!-- Please do not leave this blank -->
+
+## Related Issues
+<!-- 
+Please use this format link issue numbers: Fixes #123
+https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
+-->
```
Comment (darkmatter18) [inline .github/pull_request_template.md:21]: Add this on top of the Template

**Review by darkmatter18 — CHANGES_REQUESTED** [review]
I love your work. There are some changes, I think should be fixed.

**Comment by jason1985** [comment]
Made the changes let me know if that's ok.

Thanks!

**Pull request merged**